### ISSUE TITLE
Ship an immutable recovery mode config for local accounts

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2233]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2229
- Created at 2014-02-06 17:25:21 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:11:09 as wontfix
- Assigned to nobody
---

When SSSD manages local users, it should also ship an immutable configuration file that would be used in cases when the administrator's config doesn't work well or is not provided at all.

When implementing and testing this feature, we should make sure that the basic functionality works with this config. At least authentication and password changes must be supported.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-06 17:25:38](https://pagure.io/SSSD/sssd/issue/2229#239404)**

Fields changed

blocking:  => 2228
keywords:  => localaccounts

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-13 15:17:49](https://pagure.io/SSSD/sssd/issue/2229#239405)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.13 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-04-10 18:14:55](https://pagure.io/SSSD/sssd/issue/2229#239406)**

Fields changed

rhbz:  => todo

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-26 18:27:45](https://pagure.io/SSSD/sssd/issue/2229#239407)**

Fields changed

mark:  => 1

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-10 14:13:53](https://pagure.io/SSSD/sssd/issue/2229#239408)**

This will be important for local accounts handling, but it's out of scope for 1.13

milestone: SSSD 1.13 beta => SSSD 1.13 backlog

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-12 20:13:17](https://pagure.io/SSSD/sssd/issue/2229#239409)**

Mass-moving tickets not planned for the 1.13 release to 1.14

milestone: SSSD 1.13 backlog => SSSD 1.14 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-08-18 12:02:10](https://pagure.io/SSSD/sssd/issue/2229#239410)**

Fields changed

cc:  => mzidek@redhat.com
sensitive:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-01-26 16:22:16](https://pagure.io/SSSD/sssd/issue/2229#239411)**

When we work on this, the immutable config should also detail why there is no more elaborate config (ie we don't know what the remote domain is)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-02-16 12:11:27](https://pagure.io/SSSD/sssd/issue/2229#239412)**

The local users support won't be added in 1.14. We will need the fallback config for the secrets responder, which is tracked separately in #1552, but the providers work is not scoped for 1.14.

milestone: SSSD 1.14 beta => SSSD 1.15 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-23 12:34:44](https://pagure.io/SSSD/sssd/issue/2229#239413)**

Fields changed

owner: somebody => jhrozek
patch: 0 => 1
status: new => assigned

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 15:00:10](https://pagure.io/SSSD/sssd/issue/2229#239414)**

**Metadata Update from @jhrozek**:
- Issue assigned to jhrozek
- Issue marked as blocked by: #2228
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-28 12:10:54](https://pagure.io/SSSD/sssd/issue/2229#406229)**

* a4837791f62283079e7be4b17efb769be8b2dfd1
* 5a660d3aa67403fba69a8047ecedfe8a4276fc30

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-28 12:11:23](https://pagure.io/SSSD/sssd/issue/2229#406322)**

master:

*    a4837791f62283079e7be4b17efb769be8b2dfd1
*    5a660d3aa67403fba69a8047ecedfe8a4276fc30

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-28 12:35:36](https://pagure.io/SSSD/sssd/issue/2229#414561)**

This ticket is fixed partially because if the config cannot be loaded, we still fail
the patches only handle the 'no config' part, not the 'config is broken part'

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-28 12:39:52](https://pagure.io/SSSD/sssd/issue/2229#415378)**

This ticket is fixed partially because if the config cannot be loaded, we still fail
the patches only handle the 'no config' part, not the 'config is broken part'

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2019-08-28 15:40:51](https://pagure.io/SSSD/sssd/issue/2229#593229)**

**Metadata Update from @jhrozek**:
- Assignee reset

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-13 13:30:28](https://pagure.io/SSSD/sssd/issue/2229#633139)**

**Metadata Update from @thalman**:
- Custom field blocking reset (from 2228)
- Custom field design_review reset (from 0)
- Custom field mark adjusted to on (was: 1)
- Custom field patch adjusted to on (was: 1)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue tagged with: Canditate to close

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:11:08](https://pagure.io/SSSD/sssd/issue/2229#636620)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:11:09](https://pagure.io/SSSD/sssd/issue/2229#636621)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
